data/human/culture conversations.txt

```diff
@@ -1254,3 +1254,375 @@ mission "Little Vienna in New Austria"
+				"second place margin" >= 1
+			# 2nd place margin of 0:happens in ~9% of races, 1:~16%, 2-3:~24%,4-9:~38%,10+:~12%
+			`	They edge out the competition in a photo-finish, securing the win by a razor-thin margin.`
+				goto endingmiddle
+			label secondplaceone
+			`	They clinch victory in a breathtaking display, only a single lap ahead of the field.`
+				goto endingmiddle
+			label secondplacetwo
+			`	They rocket to an unrivaled lead in the race's last few minutes, ending the race with a clear, commanding advantage.`
+				goto endingmiddle
+			label secondplacefour
+			`	They surge ahead, their craft a blur of power and precision, leaving rivals in their wake by multiple laps to finish the race in a clear victory.`
+				goto endingmiddle
+			label secondplaceten
+			`	They dominate the competition with an astonishing double-digit lap lead to cross the finish line in an overwhelming triumph.`
+			label endingmiddle
+			branch lastplacefourteen
+				"last place margin" >= 14
+			branch lastplacenine
+				"last place margin" >= 9
+			branch lastplacesix
+				"last place margin" >= 6
+			branch lastplacethree
+				"last place margin" >= 3
+			# last place margin of 0-2:happens in ~8% of races, 3-5:~27%, 6-8:~28%,9-14:~29%,14+:~8%
+			`	The other three teams bundle together, never separated by more than two laps in the last-minute sprint for second place.`
+				goto trueending
+			label lastplacethree
+			`	The rest cluster together fairly tightly for an eventful end to the race.`
+				goto trueending
+			label lastplacesix
+			`	The mid-field hovercraft finish the race in a loose grouping over half a dozen laps apart.`
+				goto trueending
+			label lastplacenine
+			`	The remaining racers finish uneventfully spread out, the order stretched across close to a dozen laps.`
+				goto trueending
+			label lastplacefourteen
+			`	Only three racers finish today: one crashed off the track, the driver lucky to escape unharmed.`
+			label trueending
+			`	${sunracer hovercraft victory exit}`
+			`	Final results:`
+			``
+			`		Megaparsec Racing Technologies: <megaparsec name>, &[megaparsec speed] laps completed`
+			`		Syndicate Performance Racing: <syndicate name>, &[syndicate laps] laps completed`
+			`		Delta V Grand Prix: <delta name>, &[delta laps] laps completed`
+			`		Deep Sky Dynamics: <deep name>, &[deep laps] laps completed`
+			# used for testing purposes	`	2nd place: &[second place], margin: &[second place margin]; last place: &[last place], margin: &[last place margin]`
+			``
+			action
+				clear "race length"
+				clear "syndicate boost"
+				clear "delta boost"
+				clear "deep boost"
+				clear "megaparsec speed"
+				clear "syndicate speed"
+				clear "delta speed"
+				clear "deep speed"
+				clear "syndicate laps"
+				clear "delta laps"
+				clear "deep laps"
+				clear "second place"
+				clear "last place"
+				clear "second place margin"
+				clear "last place margin"
+				clear "hovercraft racing: talked to fan"
```

Comment:
Should this condition be cleared if the mission is repeatable? It seems a little weird to me that you'd get the choice to ask someone how it works every time.

```suggestion
```